chart/values.yaml

```diff
@@ -224,4 +225,15 @@ startupProbe:
     port: http
   failureThreshold: 20
   periodSeconds: 10
-  timeoutSeconds: 5
\ No newline at end of file
+  timeoutSeconds: 5
+
+
+
+extraVolumes: []
+##  - name: custom-css-data
+##    persistentVolumeClaim:
+##      claimName: custom-css-data
+
+extraVolumeMounts: []
```

Comment:
For these two new values, can you add comments in the format used above the other values? These are used to generate the table in the readme.

Reply:
I have added the comments. But I don't know how to add sample as comment for readme

Reply:
That's fine, I can generate it after this is merged. Can you fix the extra space between 'each' and 'extraVolumes'?

Reply:
see 560654d

Hi, 
i thinks, we need to give samples in commnents for extraVolumes and extraVolumeMounts